aleph0/prompts.ts

```diff
@@ -82,3 +81,39 @@ ${specificKnowledge}
 Changed snippet code:
 `;
 };
+
+export const updateFilePrompt = async (options: {
```

Comment:
Do we want to remove copy and paste here? So when we improve the prompt we improve both together.